SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -181,5 +229,52 @@ private string ExtractPreviousVersionFromChangelog(string currentLevel)
 
 			return string.Empty;
 		}
+
+		private bool SkipHeader(int index)
+		{
+			//var currentLine = _markdownLines[index];
+			bool print = false;
+			bool endOfFile = false;
+			bool currentPackage = false;
+			bool printLine = false;
+
+			//exclude unnecessary headers from changelog file
+			for (int i = index + 1; i < _markdownLines.Length; i++)
+			{
+				var line = _markdownLines[i];
+				if (string.IsNullOrEmpty(line) || line.StartsWith("  "))
+					continue;
+				if (line.StartsWith("-") && FilterEntries)
+				{
+					if (_filterRegex.IsMatch(line))
+					{
+						if (line.StartsWith($"- [{PackageId}]"))
+						{
+							currentPackage = true;
+							break;
+						}
+					}
+					else
+					{
+						printLine = true;
+						break;
+					}
+				}
+				else if (!_versionRegex.IsMatch(line))
+					break;
+
+				if (i == _markdownLines.Length - 1)
+				{
+					endOfFile = true;
+					break;
+				}
+			}
+
+			if (((currentPackage || printLine) && !endOfFile && FilterEntries) || !FilterEntries)
+			{
+				print = true;
+			}
+			return print;
```

Comment:
```suggestion
			if (!FilterEntries)
			{
				return true;
			}
			return false;
```

Reply:
Or even simpler:

```suggestion
			return !FilterEntries;
```

or 
```suggestion
			return FilterEntries ? false : true;
```